bin/lib/homepage.js

```diff
@@ -0,0 +1,149 @@
+if (process.env.NODE_ENV !== 'production') require('dotenv').config();
+const fetch = require('node-fetch');
+const { sections } = require('./page-structure');
+const structure = require('./page-structure');
+const Utils = require('./utils');
```

Comment:
ok, took me a while to get this.
Why not 
const structure = require('./page-structure');
const sections = structure.sections;

Reply:
ES6 notation